test/component-bind.spec.ts

```diff
@@ -420,7 +362,7 @@ describe('component', () => {
       expect(() => {
         @component
         class WrongComponent extends Composite {
-          @bind({path: 'foo', all: {foo: '#bar.baz'}} as any)
+          @bind({path: '#foo.bar', all: {foo: '#bar.baz'}} as any)
```

Comment:
is prepending `#` required now?

Reply:
Yeah, at some point the order of the checks apparently changed and a missing "#" now causes another error before this one.

Reply:
Do we need to update any examples/documentation to reflect that?

Reply:
Not really, it wasn't documented before. Does it matter in which order i till the user how they screwed up?

Reply:
-> not prepending `'#'` to the ID was never a valid syntax, this is all about improving the test.